.eslintrc.js

```diff
@@ -2,19 +2,19 @@ module.exports = {
   env: {
     node: true,
     mocha: true,
-    es2020: true
+    es2020: true,
   },
   extends: ["plugin:prettier/recommended", "eslint:recommended"],
   plugins: ["prettier", "mocha"],
   rules: {
     "prettier/prettier": ["error"],
-    indent: ["error", 2, { SwitchCase: 1 }],
+    indent: 0, // avoid conflict with prettier's indent system
```

Comment:
Prettier and eslint have different implementations of indent, so even when they are configured similarly, they sometimes make conflicting decisions (encountered one in affiliates), so I turned off the eslint checker to allow the prettier one to take over.